pyflakes/test/test_undefined_names.py

```diff
@@ -499,7 +515,7 @@ def fun2():
                     a
                     a = 2
                     return a
-        ''', m.UndefinedLocal)
+        ''', m.UndefinedLocal, m.UndefinedName)
```

Comment:
I don't think this new error here is correct -- it seems to be correct on the main branch